my_commends.sql

```diff
@@ -0,0 +1,63 @@
+DROP SCHEMA IF EXISTS swapbook CASCADE;
+CREATE SCHEMA IF NOT EXISTS swapbook;
+
+CREATE TABLE swapbook.users (
+    id INT PRIMARY KEY,
+    name VARCHAR (255) NOT NULL,
+    email VARCHAR (255) UNIQUE NOT NULL,
+    password VARCHAR (255) NOT NULL,
+    address VARCHAR (255)
+);
+
+CREATE TABLE swapbook.opinie (
```

Comment:
Może trzymajmy się albo angielskich nazw, albo polskich?
W reszcie sa raczej angielskie to to możne też zmienić żeby się nie myliło ?

Reply:
good point ;)

Reply:
:TODO zmień @Dominik nazwy

Reply:
@kolorowerowe  zmień nazwy wszędzie

Reply:
okej, przepisałem to narazie ze schematu bazy danych, ale już poprawiam